app/main.py

```diff
@@ -1,3 +1,63 @@
+from __future__ import annotations
+
+
 class Distance:
-    # Write your code here
-    pass
+    def __init__(self, km: int) -> None:
+        self.km = km
+
+    def __str__(self) -> str:
+        return f"Distance: {self.km} kilometers."
+
+    def __repr__(self) -> str:
+        return f"Distance(km={self.km})"
+
+    def __add__(self, other: Distance or int or float) -> Distance:
```

Comment:
You can not use `or` in type hint. Fix it here and everywhere.
```suggestion
    def __add__(self, other: Distance | int | float) -> Distance:
```